draft-fossati-tls-attestation.md

```diff
@@ -113,89 +114,37 @@ informative:
-should be defined, only that they should all be usable within our framework.
-Attestation mechanisms must be defined by companion specifications.
-
-To give the peer information that the handshake signing key is properly secured,
-the associated attestation result has to be appraised by the peer. This must be
-the case when either of the two remote attestation topologies is used. Hence,
-attestation evidence must contain the security state of both the signing key and
-of the platform hosting it. A Platform Attestation Service issues Platform
-Attestation Tokens (PAT) to prove that the Key Attestation Service has not been
-tampered with. The Key Attestation Service in turn issues Key Attestation Tokens
-(KAT) to prove that the signing key is secure, as described in {{handshake-overview}}. The
-security of the protocol critically depends on the verifiable binding between
-these two logically separate units of evidence.
+Attestation is the process by which an entity produces evidence about itself that another party can use to evaluate the trustworthiness of that entity.
+This document describes a series of protocol extensions to the TLS 1.3 handshake that enables the binding of the TLS authentication key to a remote attestation session.
+As a result, a peer can use "attestation credentials", consisting of compound platform evidence and key attestation, to authenticate itself to its peer during the setup of the TLS channel.
+This enables an attester, such as a confidential workload running in a Trusted Execution Environment (TEE), or an IoT device that is trying to authenticate itself to a network access point, to present a more comprehensive set of security metrics to its peer.
+This, in turn, allows for the implementation of authorization policies at the relying parties that are based on stronger security signals.
+
+Given the variety of deployed and emerging attestation technologies (e.g., {{TPM1.2}}, {{TPM2.0}}, {{-rats-eat}}) these extensions have been explicitly designed to be agnostic of the attestation formats.
+This is achieved by reusing the generic encapsulation defined in {{-cmw}} for transporting evidence and attestation result payloads in the TLS Certificate message.
+The proposed design supports both background-check and passport topologies, as described in {{Sections 5.2 and 5.1 of -rats-arch}}.
+This is detailed in {{evidence-extensions}} and {{attestation-results-extensions}}.
+This specification provides both one-way (server-only) and mutual (client and server) authentication using attestation credentials, and allows the attestation topologies at each peer to be independent of each other.
+
+This document does not specify any attestation technology.
+Companion documents are expected to define specific attestation mechanisms.
 
 # Conventions and Terminology
 
+The reader is assumed to be familiar with the vocabulary and concepts defined in
+{{Section 4 of -rats-arch}}, and those in {{Section 2 of -rats-kat}}.
```

Comment:
We cannot say that the use of KAT is optional and then also assume that people have read it.

Reply:
KAT is not optional, it's informative.  This is not dissimilar to -rats-arch, which is also an informative ref.

Reply:
This is one of the reasons why I'm starting to think we're overloading our KAT draft. In reality, we probably want a document talking about "architectures" or approaches to key attestation, which defines the problem and the types of solutions you might get (e.g., delegated attestation vs "integrated"?), and another document just to describe an EAT-based format. For this draft we'd only need an info reference to the former.

Reply:
This is now tracked in #41